reports/pin/pin.qmd

```diff
@@ -44,9 +45,204 @@ hie_data_subset <- hie_data %>%
+  mutate(
+    "Improvement Area" = glue("{char_bldg_sf} SqFt", .na = NULL),
+    "Property Class" = char_table_data$meta_class,
+    "Year Built" = as.character(char_table_data$char_yrblt),
+    "Elementary School District" =
+      char_table_data$loc_school_elementary_district_geoid,
+    "Secondary School District" =
+      char_table_data$loc_school_secondary_district_geoid,
+    "Bedrooms" = as.character(char_table_data$char_beds),
+    "Bathrooms" = glue("{char_table_data$char_fbath} Full /
+                       {char_table_data$char_hbath} Half", .na = NULL),
+    "Basement" = glue("{char_table_data$char_bsmt},
+                      {char_table_data$char_bsmt_fin}", .na = NULL),
+    "Garage Size" = char_table_data$char_gar1_size,
+    "Porch" = char_table_data$char_porch,
+    "Facade" = char_table_data$char_ext_wall,
+    "HVAC" = glue("{char_table_data$char_heat},
+                  {char_table_data$char_air}", .na = NULL),
+    "Roof Material" = char_table_data$char_roof_cnst,
+  ) %>%
+  help_clean_tables() %>%
+  replace(is.na(.), "-") %>%
+  gt() %>%
+  tab_options(
+    table.width = pct(100),
+    table.align = "left"
+  ) %>%
+  cols_hide(
+    columns = c(
+      char_bldg_sf, loc_school_elementary_district_geoid, meta_class,
+      loc_school_secondary_district_geoid, char_yrblt, char_beds,
+      char_bsmt, char_bsmt_fin, char_fbath, char_hbath, char_porch,
+      char_ext_wall, char_heat, char_air, char_roof_cnst, meta_card_num
+    )
+  ) %>%
+  tab_footnote(
+    footnote = "Square footage of the building, as measured from the exterior",
+    locations = cells_body(columns = "Improvement Area"),
+    placement = "right"
+  ) %>%
+  tab_footnote(
+    footnote = "Designates the property type, such as vacant, residential,
+    multi-family, agricultural, commercial, or industrial. The classification
+    determines the percentage of fair cash value at which a property is
+    assessed for taxing purposes.",
+    locations = cells_body(columns = "Property Class"),
+    placement = "right"
+  ) %>%
+  tab_footnote(
+    footnote = "Number of bedrooms in the building",
+    locations = cells_body(columns = "Bedrooms"),
+    placement = "right"
+  ) %>%
+  tab_footnote(
+    footnote = "Full bath defined as bathrooms with a shower
+    or bathtub. Half bath defined as bathroom without a shower or bathtub.",
+    locations = cells_body(columns = "Bathrooms"),
+    placement = "right"
+  ) %>%
+  tab_footnote(
+    footnote = "Basement type and finish",
+    locations = cells_body(columns = "Basement"),
+    placement = "right"
+  ) %>%
+  tab_footnote(
+    footnote = "Exterior wall construction",
+    locations = cells_body(columns = "Facade"),
+    placement = "right"
+  ) %>%
+  tab_footnote(
+    footnote = "Interior heating type/central air",
+    locations = cells_body(columns = "HVAC"),
+    placement = "right"
+  )
+```
+
 
-```{r, _pin_predicted_model_value}
+
+## Values & Sales 
+
+```{r _pin_predicted_model_value}
```

Comment:
**issue (blocking):** It looks like there's a duplicate of the same table here. Also, let's try to find a way to make the table visible without the need for scrolling.